### PR TITLE
WIP - Enterprise 4.12: Credential DPA cherry pick

### PR DESCRIPTION
**Pull request by anarnold97**
### Cherry pick for 4.12

Cherry Picked from 0b495fbf289ee49b3fd5aa190ef4c5423cdcbd43 xref: https://github.com/openshift/openshift-docs/pull/78934

### Jira

* [OADP-1443](https://issues.redhat.com/browse/OADP-1443)

### Version(s):

* OCP 4.12

Issue:

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by anarnold97** [comment]
@prasad - please can I ask you to do a quick double-check for this OCP 4.12 cherry pick for @shdeshpa07

I am pretty sure it is correct but given the different file names, it would be great if you could just quickly check

Thanks

File: modules/oadp-installing-dpa.adoc [hunk 1]
```diff
@@ -169,39 +181,39 @@ spec:
...
-    - velero:
-        provider: {provider}
-        default: true
-        config:
-          project: <project>
-          snapshotLocation: us-west1 <9>
+          bucket: <bucket_name> # <7>
+          prefix: <prefix> # <8>
+  snapshotLocations: # <9>
+          snapshotLocation: us-west1 # <10>
```
Comment (PrasadJoshi12) [inline modules/oadp-installing-dpa.adoc:201]:
```suggestion
          region: us-west1 # <10>
```

File: modules/oadp-installing-dpa.adoc [hunk 2]
```diff
@@ -113,35 +121,38 @@ spec:
...
-  snapshotLocations: <11>
+          bucket: <bucket_name> # <9>
+          prefix: <prefix> # <10>
+  snapshotLocations: # <11>
     - velero:
         config:
           resourceGroup: <azure_resource_group>
           subscriptionId: <azure_subscription_id>
           incremental: "true"
         name: default
```
Comment (PrasadJoshi12) [inline modules/oadp-installing-dpa.adoc:151]: VSL doesn't support specifying name

File: modules/oadp-installing-dpa.adoc (hunk 1 above)
Comment (PrasadJoshi12) [inline modules/oadp-installing-dpa.adoc:201]: snapshotLocation field is added twice, can you fix it ?

**Comment by anarnold97** [comment]
@PrasadJoshi12 - I have updated this and removed the old module and pointed to the `OADP 1.2 and earlier DPA`

But looking at [OpenShift Operator Life Cycles](https://access.redhat.com/support/policy/updates/openshift_operators)

image

Please can I ask if we should we add the 1.3 DPA to the assembly?

**Pull request closed without merging**